tests/validate-urls.rb

```diff
@@ -3,20 +3,25 @@
 
 require 'English'
 require 'json'
-@status = 0
+require 'httpclient'
+status = 0
 
 # Fetch created/modified files in entries/**
 diff = `git diff --name-only --diff-filter=AM origin/master...HEAD entries/`.split("\n")
 
 def curl(url)
-  `curl --fail -A "Mozilla/5.0 (compatible;  MSIE 7.01; Windows NT 5.0)" -H "FROM: https://2fa.directory" #{url}`
-  # rubocop:disable Style/GuardClause
-  unless $CHILD_STATUS.success?
-    # Break build if above cURL exited with non-zero value
-    puts "::error file=#{@path}:: Unable to reach #{url}"
-    @status = 1
-  end
-  # rubocop:enable Style/GuardClause
+  headers = { 'User-Agent' => 'Mozilla/5.0 (compatible;  MSIE 7.01; Windows NT 5.0)', 'FROM' => '2fa.directory' }
+  req = HTTPClient.new
+  req.receive_timeout = 8
+  return 0 if (res = req.get(url, nil, headers, follow_redirect: true).status == 200)
```

Comment:
```suggestion
  res = req.get(url, nil, headers, follow_redirect: true)
  return 0 if res.status==200
```
This line wasn't working as intended, as `res` was being set to a boolean, so the matching for 50x codes below couldn't find the `status` property. I think it's probably easier to read as well if this is split into 2 lines.

Reply:
I somewhat applied your suggestions as I had to rewrite some stuff due to Rubocop complaints. Let me know what you think.